### ISSUE TITLE
Add your own list

### ISSUE THREAD

**Issue by nicoursi** [jeffisaak/diceware-pass-gen#9]
Hi, i was wondering if it is possible to support adding a custom list to generate the passphrase from

**Comment by jeffisaak** [comment]
Yeah, while that's possible, I don't currently have the bandwidth to make
changes.

Best,

Jeff
